packages/formation/sass/modules/_m-megamenu.scss

```diff
@@ -2,7 +2,8 @@ $vetnav-panel-height-desktop: 490px;
 $vetnav-panel-height: 380px;
 $marketing-container-height: 380px;
 
-#mega-menu {
+#mega-menu,
+.mega-menu {
```

Comment:
In the linked PRs, do you know why `id="mega-menu"` is breaking some things?

Reply:
We want 2 instances of the mega menu, one for mobile and one for desktop. We can't use an `id` for that because ids need to be unique.